File: tradingcards-plugin/src/main/java/net/tinetwork/tradingcards/tradingcardsplugin/card/EmptyCard.java
package net.tinetwork.tradingcards.tradingcardsplugin.card;

import org.bukkit.Material;
import org.bukkit.inventory.ItemStack;

/**
 * @author sarhatabaot
 */
public class EmptyCard extends TradingCard {
    public static final ItemStack AIR = new ItemStack(Material.AIR);

    public EmptyCard() {
        super("nullCard");
    }

    @Override
    public EmptyCard get() {
        return this;
    }

    @Override
    public ItemStack buildItem(boolean shiny) {
        return AIR;
    }

}

File: tradingcards-plugin/src/main/kotlin/com/github/sarhatabaot/tradingcards/card/EmptyCard.kt
package com.github.sarhatabaot.tradingcards.card

import net.tinetwork.tradingcards.tradingcardsplugin.card.TradingCard
import org.bukkit.Material
import org.bukkit.inventory.ItemStack

/**
 * @author sarhatabaot
 */
class EmptyCard: TradingCard(
    "tc-internal-null-card",
    Material.AIR
) {
    override fun buildItem(shiny: Boolean): ItemStack = ItemStack(material)
}